// import java.io.*;
// public class ExceptionHandling5 
// {
//     public static void main(String[]args)  
//     {
//         System.out.println("Helloo");
//         try
//         {
//             FileReader file = new FileReader("data.txt");
//         }
//         catch(Exception e)
//         {
//             System.out.println("Exception HAndled");
//         }
//         catch(RuntimeException re)
//         {
//             System.out.println(RuntimException Handled);
//         }
//         System.out.println("Byeee");
//     }  
// }

//  * Output :-
//  * Unreachable catch block for RuntimeException. 
//  * It is already handled by the catch block for Exception      
//     RuntimException cannot be resolved to a variable
//     Syntax error on token "Handled", delete this token
 
